pkg/controller/notification/notification_controller_test.go

```diff
@@ -166,6 +166,50 @@ func TestNotificationDelivery(t *testing.T) {
 		require.Equal(t, "noreply@foo.com", accepted.Message.Headers.From)
 	})
 
+	t.Run("test admin notification delivery ok", func(t *testing.T) {
+		// given
+		notification := newAdminNotification("sandbox-admin@developers.redhat.com", "Alert",
+			"Something bad happened")
+		controller, request, client := newController(t, notification, ds)
+
+		// when
+		result, err := controller.Reconcile(request)
+
+		// then
+		require.NoError(t, err)
+		require.True(t, result.Requeue)
+
+		// Load the reconciled notification
+		key := types.NamespacedName{
+			Namespace: operatorNamespace,
+			Name:      notification.Name,
+		}
+		instance := &v1alpha1.Notification{}
+		err = client.Get(context.TODO(), key, instance)
+		require.NoError(t, err)
+
+		test.AssertConditionsMatch(t, instance.Status.Conditions,
+			v1alpha1.Condition{
+				Type:   v1alpha1.NotificationSent,
+				Status: corev1.ConditionTrue,
+				Reason: v1alpha1.NotificationSentReason,
+			},
+		)
+
+		iter := mg.ListEvents(&mailgun.ListEventOptions{Limit: 1})
+		var events []mailgun.Event
+		require.True(t, iter.First(context.Background(), &events))
+		require.True(t, iter.Last(context.Background(), &events))
+		require.Len(t, events, 1)
+		e := events[0]
+		require.IsType(t, &events2.Accepted{}, e)
+		accepted := e.(*events2.Accepted)
+		require.Equal(t, "sandbox-admin@developers.redhat.com", accepted.Recipient)
+		require.Equal(t, "developers.redhat.com", accepted.RecipientDomain)
+		require.Equal(t, "Alert", accepted.Message.Headers.Subject)
+		require.Equal(t, "noreply@foo.com", accepted.Message.Headers.From)
```

Comment:
Is it possible to also verify the sent email content?

Reply:
There doesn't seem to be any way to read that via the mailgun api.